src/steps/utils.js

```diff
@@ -223,3 +223,37 @@ export function toArray(v) {
   }
   return Array.isArray(v) ? v : [v];
 }
+
+/**
+ * Returns an updated value of the provided metadata value:
+ * if the metadata is a URL or an array of URLs (comma separated)
+ * and if the URL(s) contains the preview or live host,
+ * the URL(s) are replaced by the absolute URL(s) to the production host
+ * @param {PipelineState} state the request state
+ * @param {string} meta the metadata value
+ * @returns {string} meta the update metadata value or the original value
+ */
+export function makeAbsoluteURLForMeta(state, meta) {
+  if (meta && meta.startsWith('http')) {
+    const rewrite = [];
+    const split = meta.split(', ');
```

Comment:
why `, ` (with space) ? and not `/\s*,\s*/` ?

Reply:
I assume somewhere there is a `join(', ')` thus my version would be enough. But your version is more generic, I can use yours.